### DISCUSSION TITLE
how to see docs for v3 ?

### DISCUSSION THREAD

**Discussion by jt3k** [gulpjs/gulp#2485] [Help] [answered]
how to see documentation for gulp v3, using   https://gulpjs.com/ website?

I demand to add this feature! =)

**Comment by phated** [comment, marked as answer]
This is not a feature request nor an issue, so I've moved it to a Discussion.

We aren't going to put the v3 documentation on the website because it is completely unsupported now. Gulp 4 has been officially released for over 2 years and core pieces of gulp 3 aren't maintained by us, so we can't support it properly.

Going forward, we will have versioned docs on the website.

For now, you can just open a tag on GitHub and browse the documentation at that commit.
